xwiki-platform-core/xwiki-platform-annotation/xwiki-platform-annotation-rest/src/main/java/org/xwiki/annotation/rest/internal/AnnotationsRESTResource.java

```diff
@@ -69,9 +71,22 @@
     /**
      * Request parameter prefix for a filter in an annotation request.
      */
-
     private static final String ANNOTATION_REQUEST_FILTER_PARAMETER_PREFIX = "filter_";
 
+    private static final String COMMENT_KEY = "comment";
+
+    /**
+     * The name of the request parameter whose multiple values indicate the request parameters that require HTML
+     * conversion. For instance, if this parameter's value is {@code [description, content]} then the request has two
+     * parameters, {@code description} and {@code content}, requiring HTML conversion. The syntax these parameters must
+     * be converted to is found also on the request, under {@code description_syntax} and {@code content_syntax}
```

Comment:
Sounds like you encoded a lot of things in a simple String. Wouldn't that be safer for maintenance purpose to encode all that information in a dedicated component whose job would be to retrieve or update the needed information from the parameter map?